src/lt/object.cljs

```diff
@@ -349,20 +351,20 @@
     (raise inst :init)
     inst))
 
-(defn refresh! [obj]
+(defn- refresh! [obj]
   (reset! obj (update-listeners obj))
   (raise* obj (trigger->behaviors :object.instant (:tags @obj)) nil)
   (raise obj :object.refresh))
 
-(defn add-behavior! [obj behavior]
+(defn- add-behavior! [obj behavior]
```

Comment:
This is used several places outside the namespace to dynamically add behavior (also I use it in one of my plugins to add test related behaviour after an editor has been opened). I guess for symmetry rem-behavior! should also be public.